pupil_src/shared_modules/av_writer.py

```diff
@@ -190,35 +184,39 @@ def write_video_frame(self, input_frame):
             self.container.mux(packet)
         self.current_frame_idx += 1
         self.timestamps.append(input_frame.timestamp)
-        if self.audio_export:
-            for audio_packet in self.audio_rec.demux():
-                if self.audio_packets_decoded >= len(self.audio_ts):
+        if self.audio:
```

Comment:
The content of the `if` would like great as an extracted method ;) Ok, in this case it still looks ok because it's the last block in the method, but in general such high numbers of indentation can be hard to follow.

Reply:
Good point. I extracted the other functionality while I was at it.